Makefile

```diff
@@ -158,7 +158,7 @@ include network.env
 
 # Build the base command
 # WARNING: WS_ADDR is a sensitive interface and should only be exposed to trusted networks
-BASE_CMD = ./build/bin/go-quai --$(NETWORK) --syncmode full --verbosity 3
+BASE_CMD = ./build/bin/go-quai --$(NETWORK) --syncmode full --verbosity $(VERBOSITY)
```

Comment:
can you create one for syncmode as well? I guess, if you are taking this effort, do it for all the variables

Reply:
imo syncmode should always be full as there is no more light `les` mode. that would be a separate PR to remove the flag.

Reply:
https://github.com/dominant-strategies/go-quai/issues/689